rmgpy/molecule/group.py

```diff
@@ -1392,7 +1440,7 @@ def classifyBenzeneCarbons(self, partners = None):
         #check that cbfAtoms only have benzene bonds
         for cbfAtom in cbfAtomList:
             for atom2, bond12 in cbfAtom.bonds.iteritems():
-                assert bond12.isBenzene(), "Cbf atom in {0} has a bond with an order other than 'B'".format(self)
+                assert bond12.isBenzene(), "Cbf atom in {0} has a bond with an order other than 1.5".format(self)
```

Comment:
The method used to find bond orders for atoms, getBondOrderForAtom, still counts the bonds around it as 1.33. The nearby Cb atom counts the bond as 1.5.